feature/amazon/src/main/java/com/revenuecat/purchases/amazon/handler/PurchaseHandler.kt

```diff
@@ -42,21 +43,27 @@ class PurchaseHandler(
     }
 
     override fun onPurchaseResponse(response: PurchaseResponse) {
```

Comment:
is there a common base type for all of the Responses so that we can avoid code duplication?

Reply:
no... they are all different and no base class 😭